scripts/build_shapes.py

```diff
@@ -281,7 +302,7 @@ def load_EEZ(countries_codes, geo_crs, EEZ_gpkg="./data/eez/eez_v11.gpkg"):
             f"File EEZ {EEZ_gpkg} not found, please download it from https://www.marineregions.org/download_file.php?name=World_EEZ_v11_20191118_gpkg.zip and copy it in {os.path.dirname(EEZ_gpkg)}"
         )
 
-    geodf_EEZ = gpd.read_file(EEZ_gpkg).to_crs(geo_crs)
+    geodf_EEZ = gpd.read_file(EEZ_gpkg, engine="pyogrio").to_crs(geo_crs)
```

Comment:
Could be nice to use the old engine while testing this feature.
By doing this, you can test the algorithm that you are developing and not the algorithm+a new engine.

Reply:
Why would I wait 30 seconds more each time I run the file when I know of a solution?